todo/exceptions/task_exceptions.py

```
@@ -8,3 +8,9 @@ def __init__(self, task_id: str | None = None, message_template: str = ApiErrors
         else:
             self.message = ApiErrors.TASK_NOT_FOUND_GENERIC
         super().__init__(self.message)
+
+
+class UnprocessableEntityException(Exception):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)